src/WebJobs.Extensions.DurableTask/LinuxDedicatedLogger.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class LinuxDedicatedLogger : LinuxAppServiceILogger
+    {
+        private const string LoggingPath = "/var/log/functionsLogs/durableevents.log";
+
+        public override void Log(string jsonString)
+        {
+            using (var writer = new StreamWriter(LoggingPath, true))
+            {
+                // TODO: ensure this won't crash the process if an exception
```

Comment:
I'm not sure what best practices are if for some reason we lose the ability to write telemetry. On the one hand, customer crashes are bad, but on the other hand, having silent loss of telemetry is also bad. It has less impact on customers, but if the customer does have impact, we don't even know why we lost telemetry.

If we do want to avoid crashes, you can just wrap in a try-catch, and just have a comment in the catch block saying we don't want to crash the process or something along those lines.

Reply:
I'll wrap this in a try-catch and maybe we can come up with something to log on a failure as well? It would be good if we could set up a monitor that detects these failures as well

Reply:
Most guidance I've read suggests that it's better to err on the side of not crashing if logging fails. That said, we're certainly free to make our own decisions based on what we think is necessary for supporting our own product. :)

However, I'm not sure how much value adding a try/catch will add if we're going to swallow the exception anyways. It would be a bit of boilerplate to even set it up so that try/catch would work since this is a non-async method. Alternatively, we could simply add a comment here saying that we're intentionally ignoring exceptions and achieve the same outcome of clear communication of intent.

Reply:
Does an exception thrown in this layer not bubble up and cause issues higher up in the system. I guess I am unfamiliar with how uncaught exceptions (especially in non-awaited async code) actually affects the process health.

Reply:
@davidmrdavid 
> I'll wrap this in a try-catch and maybe we can come up with something to log on a failure as well? It would be good if we could set up a monitor that detects these failures as well

But where do we log the failure to log if we can't log anything? :)  I suppose it depends on whether the failure is related to writing to disk or if it's just a bug that is specific to certain logging payloads. I guess if we want to protect ourselves from those, then adding exception handling logic would be worthwhile. But just remember that we can't use `await` here, so you'd need to either add a task continuation for error handling or wrap the call in another async/await method that uses try/catch internally (you'd still need to ignore the task returned by that method, though).